walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,24 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
-}
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
+    Greeter greeter = new Greeter();
+
+    String greeting = greeter.greet("   Ada   ");
+
+    // Whitespace should be trimmed
+    Assert.assertEquals("Hello Ada", greeting);
```

Comment:
super nit: generally, the ordering should be the "result" you get goes first and what you expect goes second for assert statements. So Assert.assertEquals(greeting, "Hello Ada"); This is so you can sort of read it like "Assert that greeting (which is what you're testing) is equal to 'Hello Ada' " (which makes more sense than the other way around). Same for below.

Reply:
Ah, I see, I'll keep that in mind for future tests as well